BP/common/religion/religions/00_proto_carpathian.txt

```diff
@@ -0,0 +1,207 @@
+﻿proto_carpathian_religion = {
```

Comment:
What does this religion represent?

Reply:
Dacian Paganism in WtWSMS.

Reply:
Could it keep the name of dacian_paganism then?

Reply:
The religion family or the faith? It's supposed to be there to support illyran_pagan too if it needs/we want to add it.

Reply:
But Illyran Pagan isn't in the Carpathian region, the group doesn't make sense. They should remain separate Pagan religions.